playbook/roles/default_page/files/defaultpage-ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: page-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$1 
+spec:
+  rules:
+     - http:
+        paths:
+          - path: / 
+            pathType: Prefix
+            backend:
+              service:
+                name: defaultpage-service 
```

Comment:
If this is the default "service", I would suggest to specify it as a `defautlBackend` so that any requests that do not match a specific path, are redirected to this default page.

Reply:
I think that this annotation "nginx.ingress.kubernetes.io/rewrite-target: /$1" do the same thing, do you agree? We tested it.

Reply:
As far as I'm aware of, the Ingress controller only rewrites the path before forwarding it to the back-end services.
Did you also try and succeed in reaching the other endpoints with such an annotation (e.g. /grafana, /liqo, /prometheus, /k3sdashboard)?

Reply:
Yes, we tried only with /k3sdashboard and it works, because the others still have not changed their ingress.

Reply:
Ok, if you can reach the /k3dashboard it should be working for all the others as well.

However, I remind you to squash the commits before proceeding to merge.

I kindly ask you all (@giovannimirarchi420 @Andrew64-bit ) so have an internal sync to decide how to arrange the Ingress resource.

You can both use one single Ingress resource on which you specify all the needed /paths, rather than using an Ingress resource for specific service.

As a best practice, if not using different IngressClass, a single Ingress resource approach is recommended.

Regards,
Francesco

Reply:
Hi @fracappa, understood.
I'm waiting for the PR to be merged in order to proceed.
Thank you very much.

Giovanni